hack/tekton.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Install Tekton and required tasks in the cluster
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+export TERM="${TERM:-dumb}"
```

Comment:
The envvar is not used, right? Or is it used by some swapn?

Reply:
you mean TERM? well, test on gitaction were failing initially. Got it working after that change (imported from other hack scripts)